cla_public/templates/_phase-tag.html

```diff
@@ -13,14 +13,7 @@
 
   <p>
     <span id="aria-feedback">
-      Tell us about your experiences. Your
-      <a
-        rel="noopener noreferrer"
-        target="_blank"
-        href="https://www.surveymonkey.co.uk/r/LegalInfoJourneyMapping_CLA"
-        aria-labelledby="aria-feedback">
-        feedback<span class="visuallyhidden">(Opens in a new window)</span></a>
-        will help us continue to improve this service.</span>
+      <a href="https://checklegalaid.service.gov.uk/reasons-for-contacting">Contact us</a> if you need help or would like to give feedback to improve this service
```

Comment:
Oops, missed this one
```suggestion
      <a href="/reasons-for-contacting">Contact us</a> if you need help or would like to give feedback to improve this service
```